frontend/messages/en.json

```
@@ -53,6 +53,8 @@
 	"associatedEntityAssessments": "Associated entity assessments",
 	"associatedRepresentatives": "Associated representatives",
 	"associatedSolutions": "Associated solutions",
+	"associatedContracts": "Associated contracts",
+	"associatedEntities": "Associated entities",
```

Biome (2.1.2)

[error] The key associatedEntities was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)